x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/action/huggingface/HuggingFaceElserActionTests.java

```diff
@@ -66,6 +69,7 @@ public void shutdown() throws IOException {
         webServer.close();
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Just curious is this for the use of generics in the `PlainActionFuture`?

Reply:
No there is a (List<String>) cast in the test